docs/source/release_notes/release-notes.rst

```diff
@@ -42,65 +42,79 @@ develop
+    *    - |changed|
+         - Our dependency on immutables was bumped from 2.2.4 to 2.4.0, in order to fix an issue with static code analysis reporting errors in generated code.
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/1853>`__)
 
-    *    - |improved|
-         - ``TransactionManager`` now has an API ``getKeyValueServiceStatus`` that can return the health of the underlying KVS. This is designed for applications
-           to implement their availability status taking into account the :ref:`kvs health <kvs-status-check>`
-           (`Pull Request <https://github.com/palantir/atlasdb/pull/1832>`__)
+    *    - |devbreak|
+         - Renamed the following classes to match baseline rules. In each case, acronyms were lowercased, e.g. ``CQL`` becomes ``Cql``.
 
-    *    - |improved|
-         - Improved performance of getRange() on DbKvs. Range requests are now done with a single round trip to the database.
-           (`Pull Request <https://github.com/palantir/atlasdb/pull/1805>`__)
+              - ``CqlExpiringKeyValueService``
+              - ``CqlKeyValueService``
+              - ``CqlKeyValueServices``
+              - ``CqlStatementCache``
+              - ``KvTableMappingService``
+              - ``TransactionKvsWrapper``
 
-    *    - |devbreak|
-         - ``atlasdb-config`` now pulls in two more dependencies - the Jackson JDK 8 and JSR 310 modules (``jackson-datatype-jdk8`` and ``jackson-datatype-jsr310``).
-           These are required by the `palantir/http-remoting <https://github.com/palantir/http-remoting>`__ library.
-           This behaviour is consistent with our existing behaviour for Jackson modules (JDK 7, Guava and Joda Time).
-           If you do encounter breaks due to this addition, please contact the AtlasDB team for support.
-           (`Pull Request <https://github.com/palantir/atlasdb/pull/1810>`__)
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/1853>`__)
+
+.. <<<<------------------------------------------------------------------------------------------------------------->>>>
+
+
+======
+0.40.0
+======
+
+28 Apr 2017
+
+.. list-table::
+    :widths: 5 40
+    :header-rows: 1
+
+    *    - Type
+         - Change
 
     *    - |userbreak|
-         - AtlasDB will refuse to start if backed by Postgres 9.5.0 or 9.5.1. These versions contain a known bug that causes incorrect results to be returned for certain queries.
+         - AtlasDB will refuse to start if backed by Postgres 9.5.0 or 9.5.1.
+           These versions contain a known bug that causes incorrect results to be returned for certain queries.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1820>`__)
 
+    *    - |devbreak| |fixed|
+         - Correct ``TransactionManager.createInMemory(...)`` to conform with the rest of the api by accepting a ``Set<Schema>`` object.
```

Comment:
Normally for devbreak we tell people how to fix it, though in this case it seems pretty obvious what to do.